packages/mgt/package.json

```diff
@@ -35,7 +35,7 @@
     "compile:watch": "tsc -w",
     "lint": "eslint -c ../../.eslintrc.js 'src/**/*.ts'",
     "postpack": "cpx *.tgz ../../artifacts",
-    "rollup": "rollup -c",
+    "rollup": "rollup -c --bundleConfigAsCjs",
```

Comment:
@gavinbarron removing this option throws a Common JS error. Do you think this affects the bundle we ship?